examples/tokenvm/storage/storage.go

```diff
@@ -40,10 +40,10 @@ type ReadState func(context.Context, [][]byte) ([][]byte, []error)
 // 0x5/ (hypersdk-fee)
 
 const (
-	// metaDB
+	// VM state
 	txPrefix = 0x0
 
-	// stateDB
+	// HyperSDK state
```

Comment:
what is the difference between VM state and hyperSDK state?

Reply:
VM state is the database that is used by the implementation VM (hypersdk does not know about this) + HyperSDK state is known by hypersdk and contains data referenced by state keys. The database for the latter is defined by hypersdk but they state keys are defined by the VM

Reply:
Updated comment